package.json

```diff
@@ -46,7 +46,7 @@
     "@reduxjs/toolkit": "^1.9.2",
     "@swapr/core": "^0.3.19",
     "@swapr/periphery": "^0.3.22",
-    "@swapr/sdk": "1.8.2",
+    "@swapr/sdk": "SwaprHQ/swapr-sdk#feat/add-zksync-era-support",
```

Comment:
Change in its own commit, to be reverted once we deploy the new SDK version containing zkSync updates.

Reply:
You're right! Already fixed it and pushed the `yarn.lock` updated file for testing purposes.